digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/AssetFiller.scala

```diff
@@ -392,6 +396,59 @@ class AssetFiller {
     }
   }
 
+  private def mergeValuesExistingOnSameRoadLink(roadLink: RoadLink, segments: Seq[PersistedLinearAsset], changeSet: ChangeSet): (Seq[PersistedLinearAsset], ChangeSet) = {
```

Comment:
Since this method seems to be for Prohibitions only, can we move it and override it in a separate class? ProhibitionsFiller or something like that